client/my-sites/checkout/checkout/index.jsx

```diff
@@ -399,6 +399,11 @@ export class Checkout extends React.Component {
 			if ( isJetpackNotAtomic && ! isJetpackProduct ) {
 				return `/plans/my-plan/${ selectedSiteSlug }?thank-you&install=all`;
 			}
+			// If we just purchased a Jetpack product, redirect to the my plans page.
+			if ( isJetpackNotAtomic ) {
+				return `/plans/my-plan/${ selectedSiteSlug }`;
```

Comment:
Does it make sense to use `isJetpackProduct` for this logic, to make it more specific? It's available already. So it would become:

```
 isJetpackNotAtomic && isJetpackProduct
```

Reply:
Right now we only have isJetpackProduct and plans. 
I think it might make sense down the road but. I just wanted to keep things simple.

Reply:
That's fair. I agree it's an over-engineering with the current circumstances. 🚢